webpack.config.js

```diff
@@ -1,5 +1,17 @@
+const webpack = require("webpack");
```

Comment:
Duh yeah, I missed that (aka reviewing PRs at 1AM 😅).
Too bad we can't use `import` here though, the irony :-)